core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -3917,6 +3918,38 @@ public static List mapValues(Map map) {
     return new ArrayList<>(map.values());
   }
 
+  /** Support the MAP_FROM_ARRAYS function. */
```

Comment:
is this different/unrelated to STR_TO_MAP? if so, the commit message etc should be updated to reflect everything the PR includes.

Reply:
fix pr info and issue description